_posts/2018-08-30-numfocus-workshop.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: NumFOCUS Sponsored Workshop and Hackathon
+---
+
+We are happy to announce that thanks to a [NumFOCUS development grant][numfocus]
+we will be hosting a 2 day workshop and hackathon aimed at introducing researchers to MDAnalysis.
+The event will be free to attend and will be held at Northwestern University, Evanston IL on the 12th and 13th of November 2018.
+In addition, there are travel grants available to attendees aimed at promoting diversity in STEM.
```

Comment:
mention that the travel grants are meant to encourage people from outside Chicago to participate and women and minorities are especially encouraged to apply